src/js/App/Header/InsightsAbout.js

```diff
@@ -77,7 +77,7 @@ export class InsightsAbout extends Component {
                 { name: 'Migration Analytics', path: 'apps/migration-analytics/app.info.json', version: 'N/A' },
                 { name: 'Automation Hub', path: 'apps/automation-hub/app.info.json', version: 'N/A' },
                 { name: 'Automation Analytics', path: 'apps/automation-analytics/app.info.json', version: 'N/A' },
-                { name: 'Custom Policies', path: 'apps/custom-policies/app.info.json', version: 'N/A' }
+                { name: 'Policies', path: 'apps/custom-policies/app.info.json', version: 'N/A' }
```

Comment:
if you've updated your app's path from `custom-policies` to `policies`, you'll need to change the path here too.

Reply:
Indeed ! Fixed, thanks !
![image](https://user-images.githubusercontent.com/311855/77917811-4c128180-729b-11ea-9383-cd1ac048e586.png)